static/js/countdown.js

```diff
@@ -0,0 +1,50 @@
+var heading = document.getElementById("countdown-title");
+var daysDisplay = document.getElementById("days");
+var hoursDisplay = document.getElementById("hours");
+var minutesDisplay = document.getElementById("minutes");
+var secondsDisplay = document.getElementById("seconds");
+var startjam = new Date(Date.UTC(2018, 2, 24));
+var endjam = new Date(Date.UTC(2018, 2, 26));
+var goal;
+var now = Date.now();
+if (now+1000 >= endjam.getTime()) {
+    heading.innerHTML = "Code Jam has finished!";
+} else {
+    if (now > startjam.getTime()) {
+        heading.innerHTML = "Code Jam ends in...";
+        goal = endjam.getTime();
+    } else {
+        heading.innerHTML = "Next Code Jam starts in...";
+        goal = startjam.getTime();
+    }
+    var refreshCountdown = setInterval(function() {
+        var delta = goal - Date.now();
+        if (delta <= 1000) {
+            clearInterval(refreshCountdown);
+            location.reload();
+        }
+        var days = Math.floor(delta / (24*60*60*1000));
+        delta -= days * (24*60*60*1000);
+        var hours = Math.floor(delta / (60*60*1000));
+        delta -= hours * (60*60*1000);
+        var minutes = Math.floor(delta / (60*1000));
+        delta -= minutes * (60*1000);
+        var seconds = Math.floor(delta / 1000);
+        if (days < 10) {
+            days = "0"+days;
+        }
+        if (hours < 10) {
+            hours = "0"+hours;
+        }
+        if (minutes < 10) {
+            minutes = "0"+minutes;
+        }
+        if (seconds < 10) {
+            seconds = "0"+seconds;
+        }
+        daysDisplay.innerHTML = days;
+        hoursDisplay.innerHTML = hours;
+        minutesDisplay.innerHTML = minutes;
+        secondsDisplay.innerHTML = seconds;
+    }, 100);
```

Comment:
Is an interval of 100 needed, probably better to use 1000 and have it update every second

Reply:
I changed it to 100 because at 1000 it can get slightly out of time as executing the code takes time also, this simply reduces that error. It isn't needed though, so I can change it if you wish.

Reply:
If it reduces the error, then that's fine. Address the coverage & flake8 and it's ready